crates/torii/core/src/utils.rs

```diff
@@ -0,0 +1,11 @@
+use chrono::{DateTime, NaiveDateTime, Utc};
+
+pub fn utc_datetime_from_timestamp(timestamp: u64) -> DateTime<Utc> {
```

Comment:
```suggestion
pub fn must_utc_datetime_from_timestamp(timestamp: u64) -> DateTime<Utc> {
```

I think it is probably ok to use `expect` here since we are always parsing dates provided by the starknet rpc. We can improve naming so it isn't accidentally used in the future where that assumption doesn't hold